Sources/SourceControl/GitRepository.swift

```diff
@@ -103,9 +103,9 @@ public struct GitRepositoryProvider: RepositoryProvider {
         // FIXME: Ideally we should pass `--progress` here and report status regularly.  We currently don't have callbacks for that.
         //
         // NOTE: Explicitly set `core.symlinks=true` on `git clone` to ensure that symbolic links are correctly resolved.
-        try self.callGit("clone", "-c", "core.symlinks=true", "--mirror", repository.url, path.pathString,
+        try self.callGit("clone", "-c", "core.symlinks=true", "--mirror", repository.location.description, path.pathString,
```

Comment:
we _could_ add another accessor (via an extension or otherwise) on `RepositorySpecifier.Location` called `gitURL` or similar which would return the same value if we want to be super explicit